CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
 - A podatus followed by a virga `(eghv)` will now be kept together (no line break between the shapes),  If you would like to allow a line break there, use `(eg/hv)` instead (see UPGRADE.md and [#1045](https://github.com/gregorio-project/gregorio/issues/1045)).
 - Penalties must now be changed using the `\grechangecount` command.  See GregorioRef and [UPGRADE.md](UPGRADE.md) for details (for the change request, see [#1021](https://github.com/gregorio-project/gregorio/issues/1021)).
 - `\greemergencystretch` must now be set by changing `emergencystretch` using `\grechangedim` instead of redefining the macro.
+- `interwordspacetext` and it's related distances are now defined as fixed distances based on the `\fontdimen` values.  This means that they will scale according to the font, not the staff size.  The documenation has been updated to explain how this is done and provide the default definitions.  See [#1036](https://github.com/gregorio-project/gregorio/issues/1036).
```

Comment:
I think 99% will be puzzled by the `\fontdimen` thing, this should probably be explained a little more, like

"based on the `\fontdimen` values, so that a space between two words in a score is exactly the same as a space between two words in text if notes do not interfere"

something like that...